src/ninja.cc

```diff
@@ -38,14 +38,15 @@
 #include "browse.h"
 #include "build.h"
 #include "build_log.h"
-#include "deps_log.h"
 #include "clean.h"
 #include "command_collector.h"
 #include "debug_flags.h"
 #include "depfile_parser.h"
+#include "deps_log.h"
```

Comment:
can we keep unrelated changes out? it may affect how easily it can be backported